src/scep/Client/responses.py

```diff
@@ -24,10 +24,7 @@ def is_update_supported(self):
 
     def strongest_cipher(self):
         if self.contains(CACaps.AES):
-            if self.contains(CACaps.SHA256):
-                return 'aes256'
-            else:
-                return 'aes128'
+            return 'aes128'
```

Comment:
Can we add the other ciphers back?

Reply:
This is not correct. According to the [specification](https://www.rfc-editor.org/rfc/rfc8894.html#name-ca-capabilities-response-fo) `AES` is for `AES128-CBC` even SHA-256 is specified. In my tests if I revert this code then it does not work in our implementation

Reply:
Thanks for pointing out